.dev/tests/cypress/integration/visual-regression-tests/ahoy.spec.js

```diff
@@ -0,0 +1,20 @@
+import { getTemplateSlug } from './helpers.js';
+
+var templateSlug = getTemplateSlug(),
+    templateName = templateSlug.charAt( 0 ).toUpperCase() + templateSlug.slice( 1 );
+
+describe( 'Test the ' + templateName + ' template for visual regressions.', () => {
+  it( 'Confirm the ' + templateName + ' template has no visual regressions.', () => {
+    cy.visit( 'https://wpnux.godaddy.com/v2/?template=' + getTemplateSlug() );
+    cy.wait( 3000 ); // wait for CoBlocks gallery layouts
+
+    cy.viewport( 'macbook-15' ); // 1440x900
```

Comment:
You could create a visual regression helper to manage all of that, as it looks like you almost always do the same thing in all of your files.

```
const sizes = [
  { viewport: 'macbook-15', name: 'desktop' },
  { viewport: 'ipad-mini', name: 'ipad' },
  { viewport: 'iphone-xr', name: 'iphone' }
];

/**
 * Take screenshot of the entire page for each viewports
 * @param {string} themeSlug   Name of the theme
 */
export function captureDocument(themeSlug) {
  sizes.forEach((size) => {
      cy.viewport(size.viewport);
      cy.screenshot(`compare/${themeSlug}-${size.name}`);
    });
  });
}
```

Then lines 11 to 18 would be converted to `captureDocument( getTemplateSlug() );`

Reply:
Good call!